CHANGES.rst

```diff
@@ -20,6 +20,11 @@ hsa
 Service fixes and enhancements
 ------------------------------
 
+alma
+^^^^
+
+- Fixed a regression to handle arrays of string input for the ``query`` methods. [#2094]
```

Comment:
the number in the changelog should point to the number of the PR. I'm fixing this directly. (and this should be pulled into the dev docs highlights, too as it's also one deep in the weeds)